source/index.js

```diff
@@ -215,7 +215,7 @@ module.exports = async (input = 'patch', options) => {
 		]);
 
 		const isExternalRegistry = npm.isExternalRegistry(pkg);
-		if (!options.exists && !pkg.private && !isExternalRegistry) {
+		if (!options.availability.isAvailable && !options.availability.isUnknown && !pkg.private && !isExternalRegistry) {
```

Comment:
Shouldn't it be

```suggestion
		if (options.availability.isAvailable && !options.availability.isUnknown && !pkg.private && !isExternalRegistry) {
```

As we only want to enable 2FA in case we're publishing the package for the first time.